apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatTabsModule } from '@angular/material/tabs';
+import { RouterModule } from '@angular/router';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { LangService } from '@iglidur-designer/services';
+
+@Component({
+  selector: 'lib-toolbar',
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatIconModule,
+    MatTabsModule,
+    RouterModule,
+    TranslateModule,
+  ],
+  templateUrl: './toolbar.component.html',
+  styleUrl: './toolbar.component.scss',
+})
+export class ToolbarComponent {
+  lang!: string;
+  links = [
+    { name: 'MODEL', url: 'model' },
+    { name: 'MATERIAL', url: 'material' },
+    { name: 'POST-PROCESSING', url: 'postprocessing' },
+    { name: 'PRODUCIBILITY', url: 'producibility' },
+  ];
+  activeLink = this.links[0].name;
+
+  constructor(private translate: TranslateService, private langService: LangService) {
+    this.langService.lang$.subscribe((lang) => {
+      this.lang = lang;
+      this.translate.use('toolbar.component.i18n');
```

Comment:
Come to think of, does it mean that you will specify translation like this for every component?